spec/src/main/asciidoc/ch05-metamodel-api.adoc

```diff
@@ -149,6 +150,8 @@ must contain a declaration as follows:
 public static volatile TypedQueryReference<R> _n_;
 ----
 +
+where `n` is the name `"n"` with not a legal Java identifier characters
+replaced with an underscore.
```

Comment:
@lukasj not sure what this sentence is trying to say.

Reply:
I think you mean "with every character which is not"

Reply:
yes, that was the intent, thanks for the better language